### PR TITLE
Fix event templates to use end_date token

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Fix event templates to use end_date token

Before
----------------------------------------
Uses `$event.event_end_date`

After
----------------------------------------
Uses `{event.end_date}`  - note the conditional means only time shows when applicable

image

Technical Details
----------------------------------------

Comments
----------------------------------------

### CODE REVIEW

File: xml/templates/message_templates/event_offline_receipt_text.tpl [hunk 1]
```diff
@@ -38,7 +38,7 @@
 ==========================================================={if !empty($pricesetFieldsCount) }===================={/if}
 
 {event.title}
-{event.start_date|crmDate}{if $event.event_end_date}-{if $event.event_end_date|crmDate:"%Y%m%d" == $event.event_start_date|crmDate:"%Y%m%d"}{$event.event_end_date|crmDate:0:1}{else}{$event.event_end_date|crmDate}{/if}{/if}
+{event.start_date|crmDate}{if {event.end_date|boolean}}-{if '{event.end_date|crmDate:"%Y%m%d"}' === '{event.start_date|crmDate:"%Y%m%d"}'}{event.end_date|crmDate:"Time"}{else}{event_end_date}{/if}{/if}
```
Comment (demeritcowboy) [inline xml/templates/message_templates/event_offline_receipt_text.tpl:41]: At the end of this line: `{event_end_date}`

Comment (eileenmcnaughton) [reply xml/templates/message_templates/event_offline_receipt_text.tpl]: thanks @demeritcowboy 

Re-assuring jenkins also spotted it - CRM_Event_Form_ParticipantTest::testSubmitWithDeferredRecognition
Exception: CRM_Core_Exception: "Message was not parsed due to invalid smarty syntax : Smarty error: [in evaluated template line 41]: syntax error: unrecognized tag &#039;event_end_date&#039; (Smarty_Compiler.class.php, line 584)"
#0 [internal function](): CRM_Core_Smarty->handleSmartyError(256, "Smar

**Comment by demeritcowboy** [comment]
FYI merge conflict

**Comment by eileenmcnaughton** [comment]
Thanks @demeritcowboy  - rebased

**Comment by eileenmcnaughton** [comment]
test this please

**Comment by eileenmcnaughton** [comment]
@demeritcowboy this is passing now...

**Comment by demeritcowboy** [comment]
There's a bunch of errors but will post separately.

**Pull request merged**